.vsts-ci/linux.yml

```diff
@@ -95,24 +100,69 @@ stages:
     parameters:
       pool: ubuntu-20.04
 
+- stage: TestContainer
+  displayName: Test in a container
+  dependsOn: [BuildLinuxStage]
+  jobs:
+  - job: getContainerJob
+    displayName: Choose a container
+    pool:
+      vmImage: ubuntu-20.04
+    steps:
+    - checkout: self
+      clean: true
+
+    - checkout: Docker
+      clean: true
+
+    - pwsh: |
+        $matrix = ./PowerShell-Docker/build.ps1 -GenerateMatrixJson -FullJson -Channel preview | ConvertFrom-Json
+        $linuxImages = $matrix.preview |
+          Where-Object {$_.osversion -notlike '*windows*' -and $_.osversion -notlike '*nano*' -and $_.osversion -notlike 'alpine*' -and $_.osversion -notlike '*arm*'} |
+          Select-Object -First 100  JobName, Taglist |
+          Sort-Object -property JobName
+        $marinerImage = $linuxImages | where-object {$_.JobName -like 'mariner*'}
```

Comment:
perhaps a comment -
`# force mariner for now`

Reply:
is it possible that mariner will not be in the first 100 `$matrix.preview` after the where filter?

Reply:
```suggestion
        # Force mariner for now, later we will add automated rotation by the month.
        $marinerImage = $linuxImages | where-object {$_.JobName -like 'mariner*'}
```